pkg/controller/kubedirectorcluster/members.go

```diff
@@ -1466,8 +1561,25 @@ func appConfig(
 	if role.EventList != nil && !shared.StringInList("configure", *role.EventList) {
 		return true, nil
 	}
+
+	// Notify upgrade/rollback completion as necessary.
```

Comment:
Didn't we already send these notifications above?

Reply:
Ah, the code above probably doesn't get triggered any longer because we moved away the directory that contains the config status file. I think we can delete that older code that was intended to send the upgraded/reverted notifications.

Reply:
Ok, fixed

Reply:
Probably need to move this notification code up above the "Run the config file iff ..." comment. That code block above is an early bailout of this function if the "configure" event is not desired for this app. I guess I can imagine situations where the "configure" event is not desired but the upgraded/rollback events are.

(We should add eventlist checking for the upgraded/rolledback events too, as per issue #601, but that can be a separate PR.)

Reply:
Done